src/main/java/com/sda/games/checkers/model/Game.java

```diff
@@ -88,11 +90,13 @@ public boolean makeMove() throws Exception {
                 if (!isInputValid) {
                     System.out.println("Invalid input!");
                 } else {
-                    if (board.getBoardSpot(startX, startY) == null) {
+                    if (board.getBoardSpot(startX, startY) == null || board.getBoardSpot(startX, startY).getPiece() == null) {
```

Comment:
fajnie by bylo przeniesc to do boarda w jakiejs funkcji np board.isEmpty() czy cos w tym stylu ;)